pkg/pillar/types/locationconsts.go

```diff
@@ -46,6 +46,11 @@ const (
 	// UUIDFileName - device UUID
 	UUIDFileName = IdentityDirname + "/uuid"
 
+	// APImaybeV2 - user can statically allow to try API v2
+	APImaybeV2 = IdentityDirname + "/api-maybe-v2"
```

Comment:
I don't think we need and want dynamic behavior.
EVE 5.0 should only do V2 API.
EVE 4.10 should only do V1 APIs unless some file in /config forces it to do V2.
Hence no fallback since that just adds complexity that we do not need.

Reply:
Ok. will move the all static